openslides/core/static/js/core/site.js

```diff
@@ -525,33 +525,88 @@ angular.module('OpenSlidesApp.core.site', [
 .controller('SearchBarCtrl', [
     '$scope',
     '$state',
-    function ($scope, $state) {
-        $scope.search = function(query) {
-            $scope.query = '';
+    '$sanitize',
+    function ($scope, $state, $sanitize) {
+        $scope.search = function() {
+            var query = $sanitize($scope.querybar);
+            $scope.querybar = '';
             $state.go('search', {q: query});
         };
```

Comment:
As discussed please try to change to code in
```
        // run search with get parameter from url
        if ($stateParams.q) {
            $scope.search($stateParams.q);
            $scope.query = $stateParams.q;
        }
```
to 
```
        // run search with get parameter from url
        if ($stateParams.q) {
            $scope.query = $stateParams.q;
            $scope.search();
        }
```
Then you can remove the attribute in the search function.